Source/WebCore/page/Quirks.cpp

```diff
@@ -1570,11 +1570,32 @@ bool Quirks::needsLaxSameSiteCookieQuirk(const URL& requestURL) const
     return url.protocolIs("https"_s) && url.host() == "login.microsoftonline.com"_s && requestURL.protocolIs("https"_s) && requestURL.host() == "www.bing.com"_s;
 }
 
+#if PLATFORM(COCOA)
+
+#if !PLATFORM(IOS_FAMILY)
+static constexpr auto frozenVersion = "10_15_7"_s;
+#elif PLATFORM(WATCHOS)
+static constexpr auto frozenVersion = "11_6_1"_s;
+#elif PLATFORM(APPLETV)
+static constexpr auto frozenVersion = "18_6_2"_s;
```

Comment:
Do we need the special case for APPLETV (it seems to be the same UA than the fallback)

Reply:
We don't today, but we might in future. Might as well leave them distinct for now.

Reply:
Actually, we do! TV OS is still on 18.6 (not 18.6.2). I've corrected this.